package.json

```diff
@@ -119,6 +118,8 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "resolve-url-loader": "5.0.0",
+    "sass": "1.35.2",
```

Comment:
The [`@wordpress/scripts` installs other minor versions](https://github.com/WordPress/gutenberg/blob/fc2bf3a0c5369d5ee4f94ee73aed1feb32c357ad/packages/scripts/package.json#L85-L86) (it uses the `^`), and sass introduced a lot of deprecations because its API is changing.

I checked that Gutenberg codebase is still using it in the old way, so I also kept it using the same approach [by installing the specific version](https://github.com/WordPress/gutenberg/blob/fc2bf3a0c5369d5ee4f94ee73aed1feb32c357ad/package.json#L151-L152).

Reply:
Before checking how Gutenberg was doing it, I tried to fix all the deprecations and although risky, it's a big amount of work. So it's better to wait for Gutenberg decisions before changing it in Sensei. Notice that we also import scss files directly from Gutenberg and Calypso (which use the deprecated `@import`).